src/components/routes/Trips/ActiveTripCard.container.ts

```diff
@@ -102,14 +97,17 @@ const ActiveTripCardDesktop = styled(ActiveTripCardTablet)`
         }
       }
       .dates {
-        height: 18px;
-        width: 324px;
-        h5 {
-          ${typography('read', 2)};
-          span {
+        .date {
+          h5 {
```

Comment:
a bigger problem i see with our CSS setup is that we have to alter the default tags like `h5`

Typically a project should define sensible defaults for its styles and a programmer can just use `p, h1` tags at will without adding more CSS.
see https://getbootstrap.com/docs/4.2/content/typography/

Reply:
that sounds a lot cleaner. Maybe we can slowly transition to using default mappings? @kcvan